### ISSUE TITLE
the bugs and bullshits I found at first glance in the new unstable version.

### ISSUE THREAD

**Issue by Teals53** [openrocket/openrocket#550]
https://youtu.be/ZGoFTQtTRyk

about bugs
rail button shits
freeform shits
fit scale big shits

**Comment by Teals53** [comment]
btw why this new stage button added one more time ? its already exist here
![image](https://user-images.githubusercontent.com/13446881/66780510-a5e3e280-eed9-11e9-999d-1df8c54e0811.png)

**Comment by neilweinstock** [comment]
1. I think your fit scale problem is the same as issue #502.  Please confirm.
2. The lack of real-time feedback in the freeform fin editor is known, although I can't find which issue it's in.  If I can't track it down I'll create a new one for that.
3. I'm not sure what you're referring to with regard to "nail button".

In many cases you are asking "why" the behavior is the way it is in the unstable version.  The short answer is because... it is unstable.  There are bugs.  If you don't want to deal with these bugs then use the release version.

Also: the video demonstrating bugs is helpful.  The language is not.  Please keep it civil.

**Comment by Teals53** [comment]
1. maybe its same issue from #502 idk

2. you cant compare from 15.02 ?

3. sorry i write wrong i mean rail button :)

u say its unstable version its possible to have bugs okey but whats new in latest unstable if compare from latest release (15.02) ? i dont see any new things 

"Also: the video demonstrating bugs is helpful. The language is not. Please keep it civil."

i just cant find good word for explain this sorry xD

**Comment by neilweinstock** [comment]
1. After looking a bit more I am pretty sure they're the same.
2. Not sure what you mean.  It is a known problem in the unstable, I'm just not sure which issue # is tracking it.  If I can't find it I'll create a new one.
3.  Ah, sorry.  Yeah, the default button size is.... a bit large. :)  I'll file a new one for that.

The new features are rail buttons, boosters, and pods.  Pods are the big one.

**Comment by Teals53** [comment]
for second one i mean compare source code. rail buttons, boosters, and pods not enought for use latest unstable because latest unstable have lot of bugs etc and new features have this bugs too xD so 15.02 not have rail buttons boosters and pods but its not have this new bugs too so its more selectable version for work.

**Comment by neilweinstock** [comment]
Yes, that is why the unstable is still in development.  If you want something stable use 15.03.

**Comment by Teals53** [comment]
in my opinion latest unstable version need more features and less bugs bcs latest stable version released before 5 years ago but apparently its the opposite :/

**Comment by Teals53** [comment]
openrocket not improving itself about all of them years the opposite is getting more bad

**Comment by Teals53** [comment]
this is not normal under normal circumstances but maybe if its openrocket things change xD and its sad

**Comment by Teals53** [comment]
@teyrana all of them this bugs are fixed ? so its your closing reason if not why u closed that ?

**Comment by JoePfeiffer** [comment]
(1) An issue should be a single bug.  Not a whole collection of them.
(2) Frankly, the language in the bug report was such that I didn't bother looking at the bugs themselves.  Neil pointed this out to you.
(3) Your followups were not helpful in the slightest.  We know there are bugs, and we're working at fixing them (and there have been a *huge* number of fixes over the last couple of weeks).  Carping on the fact that an unreleased version has bugs keeping it from being doesn't do anybody any good.

In short, leaving it open wasn't useful.  We've got enough concise, clear reports to track down we didn't need this one.

**Comment by Teals53** [comment]
ok

**Comment by teyrana** [comment]
p.s. In the interest of being constructive:
https://developer.mozilla.org/en-US/docs/Mozilla/QA/Bug_writing_guidelines
